pkg/api/testapi/testapi.go

```diff
@@ -71,9 +74,16 @@ func init() {
 			internalGroupVersion: extensions.SchemeGroupVersion,
 		}
 	}
+	if _, ok := Groups["testgroup"]; !ok {
+		Groups[testgroup.SchemeGroupVersion.Group] = TestGroup{
+			externalGroupVersion: unversioned.GroupVersion{Group: testgroup.SchemeGroupVersion.Group, Version: latest.GroupOrDie(testgroup.SchemeGroupVersion.Group).GroupVersion.Version},
+			internalGroupVersion: testgroup.SchemeGroupVersion,
+		}
+	}
 
 	Default = Groups[api.SchemeGroupVersion.Group]
 	Extensions = Groups[extensions.SchemeGroupVersion.Group]
+	Testgroup = Groups[testgroup.SchemeGroupVersion.Group]
```

Comment:
Most uses is local, except the use in simple_testclient.go: https://github.com/kubernetes/kubernetes/pull/18231/files#diff-a4bd9784e18bb29a1f3024a3c4b15e8eR222, so I have to add it to the testapi map.